frontend/courses/sample/tour/events.evy

```diff
@@ -0,0 +1,62 @@
+x := 50
+y := 50
+dy := 2
+radius := 5
+stopped := false
+speed := 0.5
+hue := 0
+
+dot x y radius hue
+
+on animate
+    if stopped
```

Comment:
This is very tricksy! I was trying to figure out how an evy program actually stopped itself and wondered if there was something magical about this `stopped` variable. Turns out it was just that the run/stop button had focus so it also triggers when you press space. If you move the focus away manually and press space, the dot stops but not the program.

Just commenting.

Reply:
Hmm, good point! I always use cmd+enter to start the program these days.
I've added an unfocusRunButton function if we register the key-listener.
Let me know if you like that.